receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -40,6 +40,7 @@ type metricFamily struct {
 	labelKeysOrdered  []string
 	metadata          *scrape.MetricMetadata
 	groupOrders       map[string]int
+	exemplars         map[float64]pmetric.Exemplar
```

Comment:
Sorry I missed this in the initial review.  Exemplars should be part of the metricGroup rather than the metricFamily.

You can get the group with:

```golang
mf.getGroupKey(ls)
mg, ok := mf.groups[groupKey]
if !ok {
    ...
}

...

Reply:
Thanks, updated